packages/react-server/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+## [0.5.0] - 2019-09-11
+
+### Added
+
+- App is now wrapped with a set of default providers
```

Comment:
this not true, will update.

Reply:
Can we also document this in the readme?